plugins/verbose.ts

```diff
@@ -1,72 +1,150 @@
+import { CoreClient } from "../core/client.ts";
 import { type ClientError } from "../core/errors.ts";
 import { createPlugin } from "../core/plugins.ts";
+import { type AnyRawCommand } from "../core/protocol.ts";
 import { bold, dim, green, red } from "../deps.ts";
 
+interface RawLogPayload {
+  type: "raw_input" | "raw_output";
+  msg: string | null;
+}
+
+interface EventLogPayload {
+  type: "event";
+  event: string;
+  payload: unknown;
```

Comment:
Seems to me a little tricky to strictly type here 🤔

Reply:
Honestly, `any` might be better than `unknown` for a logger impl. Unknown makes it a tad awkward to use the variable imo.

Reply:
Whether `unknown` or `any`, I don't like them here. Ideally, I would like to have full access to all events, but at this code location, I think only core events are available.

But I agree that `any` would be less worse than `unknown` for this case.